traffic_ops/client/session.go

```diff
@@ -207,6 +244,36 @@ func LoginWithAgent(toURL string, toUser string, toPasswd string, insecure bool,
 	return to, remoteAddr, nil
 }
 
+func LoginWithToken(toURL string, token string, insecure bool, userAgent string, useCache bool, requestTimeout time.Duration) (*Session, net.Addr, error) {
+	options := cookiejar.Options {
+		PublicSuffixList: publicsuffix.List,
+	}
+
+	jar, err := cookiejar.New(&options)
```

Comment:
The build is failing because the jar variable is unused

Reply:
? It's not though... I use it on line 237

Reply:
Ah, seems I forgot something. Should work now.